### PR TITLE
Story about Rupinder Pal Singh

### PR DESCRIPTION
**Pull request by krishnappan95**

### CODE REVIEW

**Comment by krishnappan95** [comment]
Please review this, @casey-collab

File: english/Rupinderpal/rupinder.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Rupinderpal is Indian Hockey player 
+To know more about him click [here](https://en.wikipedia.org/wiki/Rupinder_Pal_Singh)
+[twitter profile](https://twitter.com/rupinderbob3?lang=en)
```
Comment (casey-collab) [inline english/Rupinderpal/rupinder.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

File: english/Rupinderpal/rupinder.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+[instagram profile](https://www.instagram.com/rupinderbob3/)
```
Comment (casey-collab) [inline english/Rupinderpal/rupinder.md]: This link is also on the line directly after another link.

File: english/marshmallow.md [hunk 3]
```diff
@@ -933,3 +933,4 @@ Life is full of [puzzles](LifePuzzle/LifePuzzle.md).
 You hear a hollow voice say "Plugh".  Mesmerized, you say "[Plugh](plugh/plugh.md)".
 
 A light appears all around you and something comes into focus in the distance. [What is that?](disney/disney.md)
+ Story about Indian Hockey Player [Rupinder Pal Singh](Rupinderpal/rupinder.md)
```
Comment (casey-collab) [inline english/marshmallow.md]: This link is also on the line directly after another link.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by krishnappan95** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :ferris_wheel:

**Pull request merged**